FitnessApp.Web/Controllers/ProductNutrientController.cs

```diff
@@ -94,6 +101,7 @@ public async Task CreateAsync([FromBody] ProductNutrientDto productNutrientDto)
             _validator.Validate(productNutrientDto, "AddproductNutrient");
 
             await _productNutrientService.CreateAsync(productNutrientDto);
+            _eventBus.Publish(new LogEvent(Statuses.Success, "Creation", productNutrientDto.GetType().Name.Replace("Dto", ""), JsonSerializer.Serialize(productNutrientDto)));
```

Comment:
productNutrientDto.GetType().Name.Replace("Dto", "")
I can't say enough: it's brilliant!

Reply:
Thx u very much! It is very important to know, that my work was appreciated.